Update service-fabric-choose-framework.md

massimobonanni opened the pull request:
There is an error on text for link "service-fabric-reliable-actors-introduction.md" at the end of the page. It must be "Reliable Actors overview", not "Reliable Services overview"

azurecla commented:
Hi __@clpolet__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (clpolet). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

massimobonanni committed:
Update service-fabric-choose-framework.md

There is an error on text for link "service-fabric-reliable-actors-introduction.md" at the end of the page. It must be "Reliable Actors overview", not "Reliable Services overview"
